### DISCUSSION TITLE
Turn off Notes in Settings

### DISCUSSION THREAD

**Discussion by myzinsky** [magicbug/Cloudlog#942] [General]
I don't need the notes. It would be nice to switch them of. Any thoughts?

**Comment by AndreasK79** [comment]
You mean to just hide the notes menu item at the top?

**Reply by magicbug** [reply]
Think so, we could add some customised options to user profile or a new section within User Profiles

**Comment by AndreasK79** [comment]
#957 added.

**Comment by magicbug** [comment]
This is now implimented

**Reply by myzinsky** [reply]
Yes it works!
